frontend/src/views/SequenceOverviewView.vue

```diff
@@ -15,7 +15,7 @@
             @click="showPopupNewSequence = true"
             v-if="allOwnSequences.length < maxSequnces || maxSequnces < 0"
           >
-            Sequenz erstellen
+            {{ $t('create', { object: $t('sequence.sequence') }) }}
           </ButtonComponent>
           <SearchBarComponent
```

Comment:
Missing placeholder, default in SearchBarComponent "Search..", no translation to german